gears/pennbbl/manifest.json

```diff
@@ -2,11 +2,11 @@
 	"name": "qsiprep-fw",
```

Comment:
Matt,
The filename needs to be 
`qsiprep-fw.json`

I will accept the pull request and do the duty.